### DISCUSSION TITLE
[Feature] Cross-instance Album Sharing

### DISCUSSION THREAD

**Discussion by cgrohs27** [immich-app/immich#12944] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The ability to share albums across different immich instances, and have them appear in the album section (not just a remote access link). It might be nice to have a "Albums on [other person's name]" section in the album list and icons for remote/local copy/both similar to how backups work in the mobile app.

Context: I am currently hosting an immich instance with my family's photos on it. My sibling is also interested in hosting their own instance for the experience and personal photos but still wants access to the shared family photos. Having to keep track of two separate instances and remembering what photo was stored where would be inconvenient. This is where the cross-instance album sharing comes in and I imagine its implementation would not look too different than the current methods for the public link and device backup features. I think a cross-instance partner sharing feature would be interesting too as I could share the entire library with them, but I would start with just albums.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#728
